### DISCUSSION TITLE
custom extension style pollution

### DISCUSSION THREAD

**Discussion by oneQiu** [ueberdosis/tiptap#5610] [General Questions]
like next-template-demo, i have more partials css。

For example, the style of the p tag is `.ProseMirror .p {fontSize: 24px}` . If the p tag is also used in my customization, the style in the customization will also change. I hope to isolate them from each other.
![image](https://github.com/user-attachments/assets/3cb88e0b-c3e6-4857-9751-3dcfd3636f7a)

How can I prevent my extension from being affected by external CSS?

**Comment by litingyes** [comment]
I create a extension to resolve this: [tiptap-extension-symbol](https://github.com/vueditor/tiptap-extension-symbol), which will add unique attr for every nodes and marks
